### ISSUE TITLE
Mapping default configuration object into parameter causes cast exception

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#1274]
**Description of the issue**

When mapping a value set a default configuration option such as `AZURE_RESOURCE_GROUP.tags` is passed as an object into parameters (i.e. `resourceGroup().tags`) an exception is raised.

**Expected behaviour**

The type should be automatically converted.

**Error output**

```text
An error occurred evaluating expression '[parameters('tags')]' line 1340. An error occurred evaluating expression '[resourceGroup().tags]' line 1308. Unable to cast object of type 'System.Collections.Hashtable' to type 'Newtonsoft.Json.Linq.JObject'."
```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.13.0-B2202103**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
